Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -25,8 +25,8 @@ import Foundation
 /// Reads and writes keychain elements that are stored on the Secure Enclave using Accessibility attribute `.whenPasscodeSetThisDeviceOnly`. The first access of these keychain elements will require the user to confirm their presence via Touch ID, Face ID, or passcode entry. If no passcode is set on the device, accessing the keychain via a `SinglePromptSecureEnclaveValet` will fail. Data is removed from the Secure Enclave when the user removes a passcode from the device.
 @objc(VALSinglePromptSecureEnclaveValet)
 @available(watchOS 3, *)
-public final class SinglePromptSecureEnclaveValet: NSObject {
-    
+public final class SinglePromptSecureEnclaveValet: NSObject, @unchecked Sendable {
```

Comment:
The localization context is a `var` on the type. It's only mutated within a lock, so we are indeed sendable. But the compiler won't be convinced of that... so we use unchecked.